ynput/ayon-core
# Max: Bug fix the "remove items" not working in Scene Inventory Manager

## Changelog Description
When the users load the objects through loaders and they want to remove them in scene inventory manager, the function doesn't work as expected to remove all data. This PR is to ensure the removal of the objects by using remove items in scene inventory manager. 

## Additional info
n/a

## Testing notes:
1. Launch Max with AYON
2. Load objects in any family
3. Go to Manage... to access Scene Inventory
4. Right Click to click "Remove items"
![image](https://github.com/ynput/ayon-core/assets/64118225/5a00194b-6643-42fb-be74-737c68f6ff3a)

5. The objects and the data would be deleted as expected.
